poll/async/get.js

```diff
@@ -1,7 +1,8 @@
 const pull = require('pull-stream')
 const sort = require('ssb-sort')
-const isPoll = require('../../isPoll')
-const isPosition = require('../../isPosition')
+const { isPoll, isPosition, isChooseOnePoll, isChooseOnePosition } = require('ssb-poll-schema')
+isPoll.chooseOne = isChooseOnePoll
+isPosition.chooseOne = isChooseOnePosition
```

Comment:
are you open to this being done on  the ssb-poll-schema side?

Reply:
My initial thought was to keep the sugar in this module.
And that I was too tired to think through how this works with depject involved.